vcx/wrappers/node/demo/faber.js

```diff
@@ -194,15 +199,20 @@ async function runFaber (options) {
     attrs: proofAttributes,
     preds: proofPredicates,
     name: 'proofForAlice',
-    revocationInterval: {}
+    revocationInterval: { to: Date.now() }
   })
 
   logger.info('#20 Request proof of degree from alice')
   await proof.requestProof(connectionToAlice)
 
   logger.info('#21 Poll agency and wait for alice to provide proof')
   let proofState = await proof.getState()
+  const revokedState = options.revocation ? StateType.None : StateType.Revoked
```

Comment:
I don't get this. I run demo and see this is how it works, but why? If I get it right:
- credential is revoked
- proof state turns from `StateType.OfferSent` to `StateType.None`
- this means it's been revoked? Is this libvcx bug?

And if `options.revocation` is `false`, then we are checking for `StateType.Revoked` but I think this branch will never be executed?